src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -141,6 +141,7 @@
     @DataBoundConstructor
     public KubernetesCloud(String name) {
         super(name);
+        setMaxRequestsPerHost(DEFAULT_MAX_REQUESTS_PER_HOST);
```

Comment:
Related?

Reply:
Just noticed that the field was set to 0 when using this constructor instead of the default value.